extensions/machine-learning/src/common/constants.ts

```diff
@@ -213,6 +213,7 @@ export const invalidModelImportTargetError = localize('models.invalidModelImport
 export const columnDataTypeMismatchWarning = localize('models.columnDataTypeMismatchWarning', "The data type of the source table column does not match the required input field’s type.");
 export const outputColumnDataTypeNotSupportedWarning = localize('models.outputColumnDataTypeNotSupportedWarning', "The data type of output column does not match the output field’s type.");
 export const modelNameRequiredError = localize('models.modelNameRequiredError', "Model name is required.");
+export const modelsRequiredError = localize('models.modelsRequiredError', "Please select at least one model to import.");
```

Comment:
@MsSQLGirl , I added this new error message and wanted to see if it looks ok to you